src/OpenTelemetry.Exporter.InMemory/InMemoryExporter.cs

```diff
@@ -23,19 +22,21 @@ public class InMemoryExporter<T> : BaseExporter<T>
         where T : class
     {
         private readonly ICollection<T> exportedItems;
-        private readonly Func<Batch<T>, ExportResult> onExport;
+        private readonly ExportFunc onExport;
 
         public InMemoryExporter(ICollection<T> exportedItems)
         {
             this.exportedItems = exportedItems;
-            this.onExport = (Batch<T> batch) => this.DefaultExport(batch);
+            this.onExport = (in Batch<T> batch) => this.DefaultExport(in batch);
```

Comment:
```suggestion
            this.onExport = this.DefaultExport;
```

Would it work to set the method to the delegate directly or does it need to be wrapped in a Func for the `in` and `out`?

Reply:
Yes! Updated